Style heading color and add label

nathanstitt opened the pull request:
Per @jason-holmes comment https://github.com/openstax/concept-coach/pull/146#issuecomment-226012959, this changes the heading back to light grey and adds the missing elements to match mockup

![screen shot 2016-06-16 at 2 49 07 pm](https://cloud.githubusercontent.com/assets/79566/16130987/7f91ee22-33d1-11e6-86c7-a2c17d57126f.png)

nathanstitt commented:
@pandafulmanda can you please review at your leisure?

pandafulmanda commented:
Looks good, except, why is `.help-text` limited to `width: 300px` when the controls are `width: 400px`?  And in the mockup, it looks like "You are joining" is ever so slightly larger than the rest of the heading.

nathanstitt commented:
great points @pandafulmanda - I've updated the sizes to be all 400px and increased the "You are joining" message size a bit.

Updated look is:
<img width="895" alt="screen shot 2016-06-17 at 6 04 53 pm" src="https://cloud.githubusercontent.com/assets/79566/16167027/3116924e-34b6-11e6-916f-d08fd9559eeb.png">
